README.md

```diff
@@ -1,8 +1,44 @@
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_BinsarHutapea/tree/master/core_java)]:
+      
+  - JDBC App: Built an application to retrieve, process, and display data from a database. PostgreSQL is used as the Relational Database Management System (RDBMS) in this application. Additionally, Docker is utilized to manage a PostgreSQL instance. Also, Maven is utilized to manage this project and its dependencies.
+  - Grep App: Implemented a tool with Java 8 for retrieving strings of text from files based on user-specified regular expressions. The program searches every existing file and subdirectory and writes any matched strings to an output file. In addition, Maven is utilized to manage this project and its dependencies. Also, this program is Dockerized and shared on Docker Hub.
+
+
+## Highlighted Projects
+**SmartProp - Blockchain-based Smart Property Ownership Management System on IPFS**: Investigated the use of Blockchain technology outside the crypto-currency realm to build Dapps, which is an application made of a smart contract and a user interface and built on a decentralized network that combines a smart contract and a frontend user interface. Built a decentralised secure online marketplace using Ethereum and other Blockchain technologies for trading properties items. Utilized Solidity for smart contract programming tasks and React framework for frond-end programming tasks. Performed integration of Dapps with IPFS to overcome the file size limitations on Blockchain and expand its user base.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2022-present)**: Develop tools and applications on a remote PC hosted on Google Cloud Platform and running on CentOS 7. Perform scripting tasks using Bash. Write queries for managing data using PostgreSQL. Manage database runtime using Docker. Handle programming tasks with Java.
+
+**Application Developer, PT Multipolar Technology Tbk (2016-2019)**: Performed web application programming and database tasks using Java, HTML, CSS, Typescript, JavaScript, Spring, Hibernate, Angular, MySQL, and Oracle. Handled receipts and reports generation using JasperReports. Wrote Selenium scripts for automated testing. Handled bug fixing. Handled live production support role. Tested web applications using manual method and automated method with Selenium. Handled application deployment and maintenance tasks using IBM Websphere Application Server and Apache HTTP Server. Wrote user manuals.
+
+
+## Education
+**Concordia University (2019-2021)**, Master of Applied Computer Science, Electrical and Computer Engineering
+
+**Gadjah Mada University (2011-2015)**, Bachelor of Computer Science, Computer Science and Electronics
+
+
+## Miscellaneous
```

Comment:
Please include a little more detail here. At least 2 items, and if you can give a few additional details about each one, that would be great.

Reply:
Thank you for the feedback. I have added a few more items to the Miscellaneous section.